lib/cli/node-modules-path.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var path  = require('path');
+
+module.exports = function(context) {
+  var nodePath = process.env.EMBER_NODE_PATH;
```

Comment:
Default `NODE_PATH` behavior is described here.  https://nodejs.org/api/modules.html#modules_loading_from_the_global_folders

If this value is unset we could actually do whatever we wanted with it, but I think doing something different than what's described in the node docs would be confusing to developers familiar with the default behavior, so keeping our own envvar seems like the best option.